src/integrationTest/java/uk/gov/hmcts/reform/amapi/controllers/AccessManagementControllerTest.java

```diff
@@ -99,7 +99,7 @@ public void testCreateResourceAccess() throws Exception {
 
         doNothing().when(accessManagementService).grantExplicitResourceAccess(any());
 
-        this.mockMvc.perform(post("/api/access-resource")
+        this.mockMvc.perform(post("/api/v1/access-resource")
```

Comment:
Please define a global "version" variable and replace all of the version hardcoding in this file

Reply:
done